[ci] remove remaining RDS dependency

suo opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #79397
* __->__ #79370

With https://github.com/pytorch/pytorch/pull/79366, the only remaining
use case for RDS is import latency stats. Afaik we do not look at these,
so removing, which eliminates our last dependency on RDS.

suo committed:
[ci] remove remaining RDS dependency

With https://github.com/pytorch/pytorch/pull/79366, the only remaining
use case for RDS is import latency stats. Afaik we do not look at these,
so removing, which eliminates our last dependency on RDS.

[ghstack-poisoned]

suo committed:
Update on "[ci] remove remaining RDS dependency"


With https://github.com/pytorch/pytorch/pull/79366, the only remaining
use case for RDS is import latency stats. Afaik we do not look at these,
so removing, which eliminates our last dependency on RDS.

[ghstack-poisoned]

suo committed:
Update on "[ci] remove remaining RDS dependency"


With https://github.com/pytorch/pytorch/pull/79366, the only remaining
use case for RDS is import latency stats. Afaik we do not look at these,
so removing, which eliminates our last dependency on RDS.

[ghstack-poisoned]

suo committed:
Update on "[ci] remove remaining RDS dependency"


With https://github.com/pytorch/pytorch/pull/79366, the only remaining
use case for RDS is import latency stats. Afaik we do not look at these,
so removing, which eliminates our last dependency on RDS.

[ghstack-poisoned]

suo committed:
Update on "[ci] remove remaining RDS dependency"


With https://github.com/pytorch/pytorch/pull/79366, the only remaining
use case for RDS is import latency stats. Afaik we do not look at these,
so removing, which eliminates our last dependency on RDS.

[ghstack-poisoned]

suo committed:
Update on "[ci] remove remaining RDS dependency"


With https://github.com/pytorch/pytorch/pull/79366, the only remaining
use case for RDS is import latency stats. Afaik we do not look at these,
so removing, which eliminates our last dependency on RDS.

[ghstack-poisoned]

suo committed:
Update on "[ci] remove remaining RDS dependency"


With https://github.com/pytorch/pytorch/pull/79366, the only remaining
use case for RDS is import latency stats. Afaik we do not look at these,
so removing, which eliminates our last dependency on RDS.

[ghstack-poisoned]

janeyx99 commented on test/test_import_stats.py:
```diff
@@ -1,74 +0,0 @@
-# as a way to track the duration of `import torch` in our ossci-metrics
-# S3 bucket (see tools/stats/print_test_stats.py)
-class TestImportTime(TestCase):
-    def test_time_import_torch(self):
```
Actually, if we do care about tracking these stats, we can just run these as normal, right? The test_runs in Rockset will keep track of these times already.

suo replied:
that's a good point! I will re-add

suo committed:
Update on "[ci] remove remaining RDS dependency"


With https://github.com/pytorch/pytorch/pull/79366, the only remaining
use case for RDS is import latency stats. Afaik we do not look at these,
so removing, which eliminates our last dependency on RDS.

[ghstack-poisoned]
